webservices/common/models/reports.py

```diff
@@ -49,11 +49,11 @@ class CommitteeReports(PdfMixin, BaseModel):
     refunded_other_political_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_OTHER_POLITICAL_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
-    total_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
+    total_contributions_period = db.Column('total_contributions_period', db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
```

Comment:
That's an excellent question actually.  When I was implementing the range filters for these sql_alchemy couldn't seem to find the column name with a default mapping.  I can circle back to that and see if these labels can actually be removed, because I had other issues at the time and I'm not entirely sure this was an issue.